Group Chat Edits (#665)
xmtp/xmtp-dot-org: docs/build/conversations.md

All the features and methods described in this document, including starting new conversations, checking if an address is on the network, and listing existing conversations, also apply to group chats. XMTP's approach to conversations is designed to be consistent across both one-on-one and group conversations, ensuring a seamless development experience.

> Refer to the [Group Chats](/docs/build/group-chat) section for more details on managing group chats specifically.

Reviewer: ```suggestion
For more details about building group chat specifically, refer to the [Group Chat](/docs/build/group-chat) documentation.
```
